cms/src/hooks/crowdin/index.ts

```diff
@@ -0,0 +1,28 @@
+import {enabled} from "./config";
+import {updateOrSetTranslationAsync} from "./source";
+import { sync } from "./sync";
+
+export type Model = "show" | "season" | "episode";
+export type Collection = "shows_translations" | "seasons_translations" | "episodes_translations";
+
+export interface Event<T> {
+    key: number;
+    keys: (number|string)[]
+    payload: T;
+    collection: Collection;
+}
+
+export default ({action, schedule}) => {
+    if (!enabled()) 
+        return
+    
+    action('items.create', updateOrSetTranslationAsync)
+    action('items.update', updateOrSetTranslationAsync)
+
+    schedule('0 * * * *', sync)
```

Comment:
Do you know how this is triggered. Because if it's in the background, then we could have many instances doing the same thing, or none at all, depending on what is currently running. IMO we should be able to trigger this via an url.